doc/contribute/doc-guidelines.rst

```diff
@@ -0,0 +1,308 @@
+.. _doc_guidelines:
+
+Documentation Guidelines for the Zephyr Project
+###############################################
+
+Zephyr Project content is written using the `reStructuredText`_ markup
+language (.rst file extension) with Sphinx extensions, and processed
+using sphinx to create a formatted standalone website.  Developers can
```

Comment:
Sphinx should be capitalized as a proper noun. (The name of the command-line tool which does the processing is sphinx-build).